### PR TITLE
Implement garbage

### PR DESCRIPTION
**Pull request by frostburn**
refs: https://github.com/frostburn/panel-league/issues/26

### CODE REVIEW

**Comment by frostburn** [comment]
This PR is submitted for initial review. Probably should be divided into separate commits.

File: lib/engine-util.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// Block colors
+const R = 'red';
```
Comment (RauliL) [inline lib/engine-util.js]: I think there should be array/object which lists all available colors in the game instead of this.

Comment (frostburn) [inline lib/engine-util.js]: The colors should be customizable in the engine. Not in the scope of this PR though.

**Comment by frostburn** [comment]
Let's merge https://github.com/frostburn/panel-league/pull/46 first.

**Pull request closed without merging**